src/cunumeric/sort/sort.cu

```diff
@@ -34,6 +34,20 @@ namespace cunumeric {
 
 using namespace Legion;
 
+class ThrustAllocator : public ScopedAllocator {
```

Comment:
You can move this to the `utils` directory, as I believe this is a common utility.

Reply:
We do not have a place for common utilities in cunumeric yet. I have created a `utilities` folder now as in legate.core.